article/app/views/liveBlogAMP.scala.html

```diff
@@ -1,7 +1,19 @@
 @import model.LiveBlogPage
+@import model.structuredData.{Organisation, LiveBlogPosting}
+@import model.LiveBlogHelpers
 
 @(model: LiveBlogPage)(implicit request: RequestHeader, context: _root_.model.ApplicationContext)
 
 @mainAMP(model, model.related, model.article.content){
+    <script type="application/ld+json">
+        @Html(Organisation().toString())
+    </script>
+    <script type="application/ld+json">
+        @Html(LiveBlogPosting(
+            model.article,
+            LiveBlogHelpers.blocksForLiveBlogRequest(model.article, request.getQueryString("page"))
```

Comment:
Unrelated but would it make more sense to only pass `model.article` and `page` request param to the `LiveBlogPosting` constructor and let it  get the blocks internally rather than exposing this `LiveBlogHelpers.blocksForLiveBlogRequest` outside of the `LiveBlogPosting` class?